rollup.config.js

```diff
@@ -53,6 +53,7 @@ export default {
       sourcemap: true,
     },
   ],
+  external: ['html5-qrcode'],
```

Comment:
just for me to understand, since it is new, why is it needed?

Reply:
So this tells Rollup not to bundle this module's code to prevent `THIS_IS_UNDEFINED` error.